tests/DCPS/DataRepresentation/DataRepresentation.cpp

```diff
@@ -17,10 +17,14 @@
 #include <dds/DdsDcpsInfrastructureC.h>
 #include <dds/DCPS/DCPS_Utils.h>
 #ifdef ACE_AS_STATIC_LIBS
-#  include <dds/DCPS/transport/tcp/Tcp.h>
+# include <dds/DCPS/transport/rtps_udp/RtpsUdp.h>
+# include <dds/DCPS/RTPS/RtpsDiscovery.h>
+# ifdef OPENDDS_SAFETY_PROFILE
+#  include <dds/DCPS/StaticDiscovery.h>
+# else
 #  include <dds/DCPS/InfoRepoDiscovery/InfoRepoDiscovery.h>
```

Comment:
Indent for preprocessor is 2 spaces like normal C++ code.
```suggestion
#  include <dds/DCPS/transport/rtps_udp/RtpsUdp.h>
#  include <dds/DCPS/RTPS/RtpsDiscovery.h>
#  ifdef OPENDDS_SAFETY_PROFILE
#    include <dds/DCPS/StaticDiscovery.h>
#  else
#    include <dds/DCPS/InfoRepoDiscovery/InfoRepoDiscovery.h>
```

Reply:
Oh, I was copying the way StaticIncludes.h did it and it had 1 space. I'll fix mine.